webapp/app/assessment/components/review/actions.ts

```diff
@@ -45,7 +45,7 @@ export const postComment =
       .then(() => {
         dispatch({ target, type: issuePostCommentCompleted, status: 'completed' })
         dispatch(getIssueSummary(countryIso, section, target))
-        dispatch(fetchCountryOverviewStatus(countryIso))
+        dispatch(CountryActions.getCountryStatus(countryIso))
```

Comment:
how come an Action is a getter? what does CountryActions.getCountryStatus(countryIso) do?

Reply:
renamed back to fetch, get as get request